rules/know-the-basic-terms-for-a-video/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Production -  Do you know the basic terms for a video?
```

Comment:
I think the title should be "Do you know the basic video terminologies?"

1. Please action

"terms" might be confused with contracts

Reply:
Done